src/components/referralform/referral.scss

```
@@ -88,12 +88,13 @@ textarea:focus
 
 .refferal-base 
```

Please correct all the misspelled words
```suggestion
.referral-base 
```